### PR TITLE
Docs updates: getMatrix(), ipGeocode(), platform and use case copy

### PR DESCRIPTION
**Pull request by nickpatrick**
Trying my hand at a docs PR!

## What?
- Update platform positioning and use case copy on Overview and API pages
- Fix Swift getMatrix() code
- Add getMatrix() to Cordova SDK docs
- Add proxy to ipGeocode() callback
- Add dots before Swift enum values

## Why?
- Reflect the latest platform positioning used on our SDK repos and solution pages
- Address https://app.clubhouse.io/radarlabs/story/6503/also-looks-like-our-ip-geocoding-docs-are-outdated-not-included-in-the-callback

### CODE REVIEW

**Comment by nickpatrick** [comment]
https://app.clubhouse.io/radarlabs/story/6503/also-looks-like-our-ip-geocoding-docs-are-outdated-not-included-in-the-callback

File: docs/sdk/cordova.mdx [hunk 1]
```diff
@@ -364,6 +364,37 @@ cordova.plugins.radar.getDistance({
...
+With the [matrix API](/api#matrix), calculate the travel distance and duration between multiple origins and destinations for up to 25 routes:
+
+```javascript
+cordova.plugins.radar.getMatrix({
+  origins: [
+    {
+			latitude: 40.78382,
+			longitude: -73.97536
+		}
+  ],
```
Comment (andlum) [inline docs/sdk/cordova.mdx:378]: {nit} Spacing looks a little off here:
image

**Review by andlum — APPROVED** [review]
Had 1 minor nit, but otherwise looks good!

File: docs/sdk/cordova.mdx (hunk 1 above)
Comment (nickpatrick) [reply docs/sdk/cordova.mdx:378]: Good catch, looks like I pasted in some tabs instead of spaces, fixing.

Nits encouraged in docs PRs!!!

**Comment by andlum** [comment]
Nice! Looks good, merging~

**Pull request merged**